### PR TITLE
Use conda to install shap

### PR DESCRIPTION
**Pull request by seanpmorgan**
*Issue #, if available:*
Fixes https://github.com/aws/amazon-sagemaker-examples/issues/1890

*Description of changes:*
Installs conda without gcc available outside of conda.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: autopilot/model-explainability/explaining_customer_churn_model.ipynb [hunk 1]
```diff
@@ -90,7 +90,7 @@
    "metadata": {},
```
Comment (hsl89) [inline autopilot/model-explainability/explaining_customer_churn_model.ipynb:90]: <p>What's the benefit of it over the existing one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (hsl89) [inline autopilot/model-explainability/explaining_customer_churn_model.ipynb:90]: <p>It should not be that involved to write a function, call it <code>create_automl_job</code>  that creates an automl job on a simple dataset and return the job id.  So we can do something like</p><p> </p><pre class="ql-syntax" spellcheck="false">automl_job_name = None # replace None by your autopilot job that exists

if automl_job_name == None:
  automl_job_name = create_automl_job()
 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (seanpmorgan) [reply autopilot/model-explainability/explaining_customer_churn_model.ipynb:90]: If you're referring to the job name `<` and `>` fail the regex validation

Comment (seanpmorgan) [reply autopilot/model-explainability/explaining_customer_churn_model.ipynb:90]: Will add this in a bit

**Pull request merged**